leader-election-api/src/main/java/com/palantir/leader/PingableLeader.java

```diff
@@ -55,5 +68,21 @@ public interface PingableLeader {
     @Path("pingV2")
     @Produces(MediaType.APPLICATION_JSON)
     @Timed
+    @Handle(method = HttpMethod.GET, path = "/leader/pingV2")
     PingResult pingV2();
+
+    enum TextPlainSerializer implements SerializerFactory<String> {
```

Comment:
See for another example https://github.com/palantir/atlasdb/blob/87f6c8eaba466579bbb04d36674224746f63b1e0/timelock-impl/src/main/java/com/palantir/atlasdb/timelock/TimestampManagementResource.java#L96